llmfoundry/data/packing.py

```diff
@@ -183,6 +211,7 @@ def _first_fit_bin_packing(
     num_bins: int,
     max_bin_size: int,
     existing_bins: List[Tuple[int, Dict[str, torch.Tensor]]],
+    is_profiling: bool = False,
```

Comment:
I think it'll make more sense to have this as a class method and just use `self.is_profiling`

Reply:
Similar reason to wanting to rename the is_profiling param in Seq2SeqCollator. It doesn't make a lot of sense if you look at this function individually.

Reply:
Done